test/test_scanner.py

```diff
@@ -392,19 +392,15 @@ def _file_test(self, url, filename, package, version):
                 ],
             ),
             (
-                "test-openssh-7.9.out",
-                "openssh-client",
-                "7.9",
+                "test-openssh-7.7p1.out",
+                "openssh",
```

Comment:
It may be somewhat counterintuitive, but the openssh checker never actually returns `openssh` -- it always returns either `openssh-server` or `openssh-client` so this test isn't ever going to work.  Also, please don't change existing tests that are working: you should add your new test below instead.